BootloaderCommonPkg/Include/Guid/LoaderPlatformDataGuid.h

```diff
@@ -16,9 +17,14 @@ extern EFI_GUID gLoaderPlatformDataGuid;
 
 #define  DEBUG_LOG_BUFFER_SIGNATURE         SIGNATURE_32 ('D', 'L', 'O', 'G')
 
-#define  DEBUG_OUTPUT_DEVICE_LOG_BUFFER     BIT0
-#define  DEBUG_OUTPUT_DEVICE_SERIAL_PORT    BIT1
-#define  DEBUG_OUTPUT_DEVICE_CONSOLE        BIT2
+//
+// Debug Interfaces
+//
+#define  DEBUG_DEVICE_RAM                   BIT0
+#define  DEBUG_DEVICE_ISA_SERIAL            BIT1
+#define  DEBUG_DEVICE_USB                   BIT3
```

Comment:
I don't think we should have ISA_SERIAL and SERIAL I/O.
And I don't think we should remove console device.

Reply:
There are several UPDs related with ISA serial and PCH serial. Had better have a common way for user to set which one is expected to use.  here we listed debug device interface, console doesn't belong to debug device.

Reply:
In SBL, ISA UART and PCH UART are handled in the same way.  We don't need a special type for each of them.  FSP might have its own definitions, we don't have to change it to match FSP.  Instead, platform code can do necessary remapping to set the FSP UPD properly.
Console debug device is very special for SBL.  FSP will not do it, SBL does support this feature and should not be removed.   It is to redirect debug output to console device.   EX:  Frame buffer is console,  we can redirect debug to frame buffer even though we don't have DebugLib for frame buffer device.